Plattform/src/components/Stackblitz/StackblitzContainer.tsx

```diff
@@ -60,49 +68,37 @@ const StackblitzContainer = ({ path, open }: Pick<StackblitzProps, 'path'> & { o
                 mounted = false
             }
         }
+        if (actualLecture) {
+            embededPromise = StackBlitzSDK.embedProject(
+                path,
+                {
+                    files: actualLecture.files,
+                    description: '',
+                    title: '',

```

Comment:
können wir dem Nutzer eine Beschreibung und / oder einen Titel anzeigen?

Reply:
man kann die description des nodes anzeigen. 
wie man den Titel anzeigen kann habe ich nicht herausgefunden. Allerdings steht der auch mindestens in den Breadcrumbs.

Reply:
wird nach Absprache nicht gemacht, da nicht notwendig und zu viel Text im iFrame